news/2019-06-15-new-featured-artist-riran.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "New Featured Artist: RiraN"
+date: 2019-06-15 06:00:00 +0000
+---
+
+osu!'s latest Featured Artist enters the scene in style - hardstyle infact! Meet [**RiraN**](https://osu.ppy.sh/beatmaps/artists/40), one of Asia's up and coming hardstyle artists!
+
+![](https://assets.ppy.sh/artists/40/header.jpg)
+
+We're dedicated towards ensuring a variety of musical styles remain represented in our choice of upcoming Featured Artists, and **RiraN** leads the charge into a sound that many rhythm gamers know and love - hardstyle.
+
+Featured across many other rhythm games, music competitions and compilation albums, **RiraN** has been gradually gaining steam since 2015 in the scene. We're proud to introduce his work to osu! with a selection of **nine** new tracks featuring his signature hardstyle hits, all neatly bundled into less than three minutes per piece.
+
+Interested? Check out a sample of a few tracks below:
+
+### Unshakable
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/40/previews/950.mp3" type="audio/mpeg">
+</audio>
+ 
+### Take The Long Way Home
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/40/previews/951.mp3" type="audio/mpeg">
+</audio>
+
+### Ready For The Madness
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/40/previews/948.mp3" type="audio/mpeg">
+</audio>
+
+### Full Control
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/40/previews/945.mp3" type="audio/mpeg">
+</audio>
+
+### Explosion
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/40/previews/944.mp3" type="audio/mpeg">
+</audio>
+
+---------
+
+All of the above plus more are now available over at [**RiraN**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/40), 100% free for use in mapping.
+
+The [Mappers' Guild](http://mappersguild.com/) will also be hosting a new series of quests based on these new tracks in the coming days, so if you're a member of the Guild and fancy a stab at some hardstyle mapping, start preparing! If you're not yet a member of the Guild - get three ranked maps to your name and come aboard.
+
+If this sort of sound is your jam, never fear - we've got more coming at the end of the month. A LOT more. The observant among us may have already caught wind of a hint we dropped at the beginning of the month!
+
+As an extra note, you may see a few more releases from us over the coming weeks. Both pishifat and Toy have been hard at work securing a *gigantic* roster of new talent from a variety of genres and musical styles both familiar and fresh that we're super hyped to share with you guys. Keep your eyes and ears peeled!
```

Comment:
Can remove the first "both" since you use it later in the same sentence

Profile link pishi (https://osu.ppy.sh/users/3178418) and Toy (https://osu.ppy.sh/users/2757689)